scripts/populate-instance.sh

```diff
@@ -113,22 +113,24 @@ ${INVENIO_WEB_INSTANCE} users create \
 
 # sphinxdoc-populate-with-demo-records-begin
 # discover the location of demo MARC21 record file:
+
 demomarc21pathname=$(echo "from __future__ import print_function; \
 import pkg_resources; \
 print(pkg_resources.resource_filename('invenio_records', \
   'data/marc21/bibliographic.xml'))" | python)
 
+SCHEMA="http://${INVENIO_WEB_HOST}:5000/schema/marc21/bibliographic/bd-v1.0.0.json"
+
 # convert demo records from MARC21 to JSON and load them
 # using randomly generated UUIDs:
-demouuids=$(dojson -i $demomarc21pathname -l marcxml do marc21 | \
-             ${INVENIO_WEB_INSTANCE} records create)
+demouuids=$(dojson -i $demomarc21pathname -l marcxml do marc21 schema ${SCHEMA} | \
+            ${INVENIO_WEB_INSTANCE} records create --pid-minter recid)
 # sphinxdoc-populate-with-demo-records-end
 
-# sphinxdoc-register-pid-begin
-recid=1
-for demouuid in $demouuids; do
-    ${INVENIO_WEB_INSTANCE} pid create \
-         -t rec -i $demouuid -s REGISTERED recid $recid
-    let recid=recid+1
-done
-# sphinxdoc-register-pid-end
```

Comment:
please find the section `# sphinxdoc-register-pid-begin` in documentation and update the text around mentioning `--pid-minter recid`